XSUP-53761 !FormatURL command causing the URL to get corrupted (#40733)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_19_92.md

#### Scripts

##### FormatURL

> - Fixed an issue where the FormatURL script would not format correctly URLs with a hex-encoded slashes in their path.

Reviewer: ```suggestion
- Fixed an issue where the FormatURL script would not format correctly URLs with hex-encoded slashes in their path.
```

> - Updated the FormatURL script to fix an issue where the script would not correctly format URLs with a hex encoded backslash in the path.

Reviewer 2: ```suggestion
- Fixed an issue where the FormatURL script would not format correctly URLs with a hex-encoded backslashes in their path.
```
